src/engraving/compat/midi/midirender.cpp

```diff
@@ -351,7 +351,7 @@ static void collectNote(EventMap* events, const Note* note, CollectNoteParams no
 
     NoteEventList nel = note->playEvents();
     size_t nels = nel.size();
-    for (int i = 0, pitch = note->ppitch(); i < static_cast<int>(nels); ++i) {
+    for (size_t i = 0; i < nels; ++i) {
```

Comment:
@Jojo-Schmitz why those are no longer creating warnings?
And does "for (int = 0, pitch = ... )" create them or you changed just because it's being used just once below?

Reply:
Maybe they never did, not  sure, but while `pitch` needs to be an `int`, i` should not

Reply:
And yes, `note->pitch` is only used once, so IHMO no need to create a separate variable